src/drawers/areaDrawer.ts

```diff
@@ -16,7 +16,7 @@ export module Drawers {
     }
 
     public selectionForIndex(index: number) {
-      return this.renderArea().select(this.selector());
```

Comment:
The more that I use `d3.select` the more that I get scared.  Can we instead use `this.renderArea()` instead of `d3`?